### ISSUE TITLE
TypeError: Cannot read property '_command' of undefined

### ISSUE THREAD

**Issue by petetnt** [sprintr/brackets-color-palette#9]
Saw this today, something has been deprecated in `1.8`  I think

Steps to repro:

1. Right click on the file tree
2. Error gets thrown

```
 Exception in 'beforeContextMenuOpen' listener on ContextMenu {id: "project-context-menu", _eventHandlers: Object, constructor: function, parentClass: Menu, on: function…} TypeError: Cannot read property '_command' of undefined TypeError: Cannot read property '_command' of undefined
    at removeMenuItemEventListeners (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/command/Menus.js:186:17)
    at ContextMenu.Menu.removeMenuItem (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/command/Menus.js:467:9)
    at file:///C:/Users/Pete/AppData/Roaming/Brackets/extensions/user/io.brackets.color-palette/main.js:330:15
    at ContextMenu.trigger (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/utils/EventDispatcher.js:229:40)
    at ContextMenu.open (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/command/Menus.js:1068:14)
    at HTMLDivElement.<anonymous> (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/command/DefaultMenus.js:311:27)
    at HTMLDivElement.n.event.dispatch (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/thirdparty/jquery-2.1.3.min.js:3:6444)
    at HTMLDivElement.r.handle (file:///C:/Program%20Files%20(x86)/Brackets/dev/src/thirdparty/jquery-2.1.3.min.js:3:3219)ErrorNotification.js:122 window.console.errorEventDispatcher.js:231 triggerMenus.js:1068 ContextMenu.openDefaultMenus.js:311 (anonymous function)jquery-2.1.3.min.js:3 n.event.dispatchjquery-2.1.3.min.js:3 r.handle
```

**Comment by sprintr** [comment]
@petetnt Sorry for late reply, I'll fix it for the next release.

**Comment by sprintr** [comment]
Fixed in v3.0.0
